app/src/main/java/com/amaze/filemanager/filesystem/root/ListFilesCommand.kt

```diff
@@ -50,24 +53,39 @@ object ListFilesCommand : IRootCommand() {
         onFileFoundCallback: (file: HybridFileParcelable) -> Unit
     ) {
         val mode: OpenMode
-        if (root && !path.startsWith("/storage") && !path.startsWith("/sdcard")) {
+        var _path = path
+        if (root &&
+            _path.equals(
+                    File(
+                            Environment.getExternalStorageDirectory(),
+                            "Android/data"
+                        ).absolutePath
+                ) &&
+            Build.VERSION.SDK_INT > Q
+        ) {
+            // Using root hack for Android >= 11 to access Android data directory
+            _path = "/data/media/0/Android/data"
```

Comment:
It would be nice if we support multiple users by passing the actual user ID

Reply:
Come to think of it, using `/data/media` isn't even needed if we have root, so we don't need to rewrite the path.

The reason my workaround was needed is because I needed to avoid `/sdcard` in order to trick Amaze into using the root codepath. Also, I haven't tested whether my workaround works on devices with actual SD cards

Reply:
Seems SD card won't be visible via `/data/media` anyway.